armi/runLog.py

```diff
@@ -503,9 +515,41 @@ class RunLogger(logging.Logger):
         :id: I_ARMI_LOG
         :implements: R_ARMI_LOG
 
+        This logger makes it easy for users to add log statements to
+        and ARMI application, and ARMI will control the flow of
+        those log statements. In particular, ARMI overrides the normal
+        Python logging tooling, to allow developers to pipe their log
+        statements to both screen and file. This works for stdout and
+        stderr.
+
+        The log statements themselves can be any text a user wants to
+        record during a run. For instance, basic notifications of what
+        is happening in the run, or basic warnings about something that
+        could be an issue, so you want people to know about. Likewise,
+        full errors can be recorded with this tool. These different
+        types of log statements are easily identified by specifying the
+        "verbosity" of the log statement, in the code where you do the
+        logging. The most important verbosities are: error, warning,
+        info (which is just to add information to the log), and debug
+        (which is meant to be extremely detailed and not always
+        desired).
```

Comment:
I would think calling out `_RunLog.setVerbosity` would be needed here? The verbosity is set that way. 

And maybe the full list of available log levels/types are in `_RunLog.getLogLevels`.

Hard to call those out since they are in a private class, but would textually doing it be OK even if the API-linking wouldn't happen?

Reply:
I will add some text that shows how end-users will call the logger:

```python
runLog.error("This sort of error might usually terminate the run.")
runLog.warning("The run will not terminate, but users might want to know.")
runLog.info("This is the usual verbosity.")
runLog.debug("This is only logged during a debug run.")
```